hack/testdata/auth/testcert.csr

```diff
@@ -0,0 +1,15 @@
+-----BEGIN CERTIFICATE REQUEST-----
```

Comment:
For all of these new testdata files, add:

1. A comment describing how they were generated 
2. A note saying that this is a test-only credential and that no security vulnerabilities should be reported in regards to them being published to a public repo